package.json

```diff
@@ -51,7 +51,7 @@
     "react-icons": "^4.7.1",
     "react-lite-youtube-embed": "^2.4.0",
     "rudder-sdk-js": "^2.25.0",
-    "sharp": "^0.32.0",
+    "sharp": "^0.32.6",
```

Comment:
Low LOGAF.

Does this library version update have anything to do with the rest of the PR?

Reply:
This was changed when you were trying to make the ui better.